KSScreenshotManager requires to limit iPhoneSimulator archs to 'i386'

The current version of `KSScreenshotManager` uses a symbol only available in the 32bit simulator:
```
Undefined symbols for architecture x86_64:
  "_UIGetScreenImage", referenced from:
      -[KSScreenshotManager saveScreenshot:includeStatusBar:] in libPods-KSScreenshotManager.a(KSScreenshotManager.o)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
rake aborted!
```

So unless the build architectures for the simulator are explicitly limited to `i386` the above linker error will occur. The issue was reported - and so far dismissed - (see: https://github.com/ksuther/KSScreenshotManager/issues/18)

Currently I'm working around this by having the following hack in my Rakefile:

```ruby
app.development do
    # ...
    if Rake.application.top_level_targets.include? 'screenshots'
        app.archs['iPhoneSimulator'] = %w(i386)
    end
    # ...
end
```
With this I can run `rake screenshots` and the screenshots are created and saved in the applications documents directory. At this point issue #3  kicks in and prevents them from being copied to my projects screenshots/ directory...

--- a/lib/motion-screenshots.rb
+++ b/lib/motion-screenshots.rb
@@ -3,6 +3,7 @@ unless defined?(Motion::Project::Config)
 end
 
 require 'motion-cocoapods'
+require 'motion-env'
 require 'fileutils'
 require 'shellwords'
 
@@ -39,6 +40,12 @@ namespace 'screenshots' do
 
     app_config.is_taking_screenshots = true
     app_config.env['MOTION_SCREENSHOTS_RUNNING'] = true
+
+    if app_config.archs['iPhoneSimulator'].include? 'x86_64'
+      # required until KSScreenshotManager is 64bit compatible
+      App.warn 'Forcing 32bit-only build target for screenshots..'
+      app_config.archs['iPhoneSimulator'] = %w(i386)
+    end
 
     screenshots_output_path = ENV['SCREENSHOTS_DIR']
     screenshots_output_path ||= App.config.screenshots_output_path

--- a/motion-screenshots.gemspec
+++ b/motion-screenshots.gemspec
@@ -19,8 +19,8 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "motion-cocoapods", ">= 1.4.0"
-  spec.add_dependency "motion-env", ">= 0.0.3"
+  spec.add_dependency "motion-cocoapods", ">= 1.7.0"
+  spec.add_dependency "motion-env", ">= 0.0.4"
 
   spec.add_development_dependency "rake"
 end

